python/lsst/daf/butler/cli/butler.py

```diff
@@ -49,8 +49,19 @@ def cmdNameToFuncName(commandName):
 
 class LoaderCLI(click.MultiCommand):
 
+    def __init__(self, *args, **kwargs):
+        self.commands = None
+        super().__init__(*args, **kwargs)
+
     @staticmethod
     def _getPluginList():
+        """Get the list of importable packages that contain butler commands.
+
+        Returns
+        -------
+        list of string
```

Comment:
Returns are supposed to have a variable name and follow the same format as parameters. Also there are multiple ways of writing this type but
```
`list` [`str`]
```
would be preferable to "list of string".  See https://developer.lsst.io/python/numpydoc.html?highlight=numpydoc#sequence-types